docs/en/revocation_endpoint.rst

```diff
@@ -5,12 +5,12 @@ Revocation Endpoint
 
 An RP MAY request the revocation of an Access Token or a Refresh Token issued by an OP.
 
-When the user logs out, the RP MUST revoke the Access Token in its possession.
+When the user logs out or when his/her session with the RP expires (according to the policies decided by the RP) the RP MUST revoke the Access Token in its possession.
 
 .. note::
   The Access Token revocation implies revoking all the Refresh Tokens linked to it.
 
-The OP will have to revoke the token specified in the request and will have to end the Single Sign-On session, if it is still active.
+The OP will have to revoke the token specified in the request and will have to end the Single Sign-On session, if it is still active. Any other active tokens for the user must be kept valid.
```

Comment:
is Single Sign-On a defined term?

would we like to define SSO in the defined terms to give our assumption on its meaning?

Reply:
Questi contenuti andranno gestiti con box di disambiguazione sulla base anche della PR #131

Reply:
Nel prossimo meeting dovremmo affrontare il problema della definizione di "sessione di SSO" e di "SSO".